### ISSUE TITLE
[DependencyInjection] Circular reference detected for service "routing.loader", path: "routing.loader" 

### ISSUE THREAD

**Issue by phansys** [symfony/symfony#26780]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.4.7

After updating from 3.4.6 to 3.4.7, following exception is thrown while `cache:clear` execution if `AsseticBundle` is enabled:
```
[Symfony\Component\DependencyInjection\Exception\ServiceCircularReferenceException]  
  Circular reference detected for service "routing.loader", path: "routing.loader".
```
Example using `symfony/framework-standard-edition`: https://github.com/phansys/symfony-standard/commit/f6b3c3b32935f096e48f9bead6a2c6d6f2d920e6.

**Comment by tomhv** [comment]
This also happened to me, but since I don't need `AsseticBundle` I've removed it and it's working fine now.

**Comment by xabbuh** [comment]
see #26755 which was fixed in #26758 and will be part of the next release
